src/rules/order.js

```diff
@@ -770,7 +770,7 @@ function makeNewlinesBetweenReport(context, imported, newlinesBetweenImports_, n
 
       if (shouldAssertNewlineBetweenGroups) {
         if (currentImport.rank !== previousImport.rank && emptyLinesBetween === 0) {
-          if (distinctGroup || !distinctGroup && isStartOfDistinctGroup) {
+          if (distinctGroup && isStartOfDistinctGroup) {
```

Comment:
This would cause breakage unless it's:
```suggestion
          if (distinctGroup || isStartOfDistinctGroup) {
```

Reply:
Hah, you're right.